core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -94,9 +99,9 @@
   public ExpectedException mThrown = ExpectedException.none();
 
   @ClassRule
-  public static ManuallyScheduleHeartbeat sManuallySchedule = new ManuallyScheduleHeartbeat(
-      HeartbeatContext.MASTER_TTL_CHECK,
-      HeartbeatContext.MASTER_LOST_FILES_DETECTION);
+  public static ManuallyScheduleHeartbeat sManuallySchedule =
```

Comment:
as a rule of thumb do not reformat code that you do not modify ... it makes it harder to review the PR